langchain/src/chains/llm_chain.ts

```diff
@@ -86,19 +86,35 @@ export class LLMChain<T extends string | object = string>
     return finalCompletion;
   }
 
+  /**
+   * Run the core logic of this chain and add to output if desired.
+   *
+   * Wraps _call and handles memory.
+   */
+  call(
+    values: ChainValues & this["llm"]["CallOptions"],
```

Comment:
My ideal signature here (when v1 comes around?) would be:
```
call(
  values: ChainValues
  options: this["llm"]["CallOptions"] // Includes "callbacks?"
)
```
So as not to have any restricted/meta properties as I'm sure we'll keep adding more and more. Would that make sense long-term?

Otherwise, if this is the long-term solution, could it make sense to make just `options` or `callOptions` (and I guess `stop`) a special value within `values` here?
```
call(
  values: ChainValues & {
    options: this["llm"]["CallOptions"]
  }
)
```
I see you have something like this below for `openai-chat` but building it in lower level might make sense to me if this is the long-term fix.

Reply:
I think it could make sense to add callbacks in call options

Reply:
Yea, the thing is only language models have call options, whereas lots of other objects in the library have callbacks, so not sure about that